python/python-core/src/main/python/domain/_annotations.py

```diff
@@ -244,13 +244,14 @@ class CascadingUpdateShadowVariable(JavaAnnotation):
     Notes
     -----
     Important: it must only change the shadow variable(s) for which it's configured.
+    It is only possible to define either `sourceVariableName` or `sourceVariableNames`.
     It can be applied to multiple attributes to modify different shadow variables.
     It should never change a genuine variable or a problem fact.
     It can change its shadow variable(s) on multiple entity instances
     (for example: an arrival_time change affects all trailing entities too).
 
-    Examples
-    --------
+    Example - Single source
+    ------------------------
```

Comment:
```
Examples
--------
```

is a standard section used in NumPy formatted docs. Do not change it or add new sections. See https://numpydoc.readthedocs.io/en/latest/format.html .

For this, it would be 

```
Examples
--------

Single source

(example here)

Multiple sources

(example here)
```